### PR TITLE
Add PyPI releases

### LINKED ISSUE #31: add version 0.1?
Could be good to release a version/ tag - so you still know which version you used for the paper.

### PR DESCRIPTION
**Pull request by znicholls**
Closes #31 

- [x] Requires #78

I reserved the name with an alpha 0.1.0 release (see https://pypi.org/project/mesmer-emulator/).

The rest of this sets up the infrastructure for future releases. @mathause what's your pypi username? @leabeusch do you want to do pypi management? If yes, please also post your pypi username.

### CODE REVIEW

**Comment by znicholls** [comment]
Once mesmer is on PyPI, it's trivial to add it to conda

**Comment by mathause** [comment]
My pypi name is also mathause. Thanks for setting this up!

**Comment by mathause** [comment]
We will have to update the installation instructions as well.

**Comment by znicholls** [comment]
Will do conda release at the same time (see https://github.com/conda-forge/staged-recipes/pull/15574)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - Changed: changes in existing functionality
+    - Deprecated: soon-to-be removed features
+    - Removed: now removed features
+    - Fixed: any bug fixes
+    - Security: in case of vulnerabilities.
+
+v0.1.0b0 - 2021-07-09
+---------------------
+
+- (`!79 <https://github.com/MESMER-group/mesmer/pull/79>`_) First release on PyPI and conda
```
Comment (mathause) [inline CHANGELOG.rst:20]:
```suggestion
- (`#79 <https://github.com/MESMER-group/mesmer/pull/79>`_) First release on PyPI and conda
```
we could also do ``:pull:`220` `` using `sphinx.ext.extlinks` but this only renders on rtd.

File: doc/source/installation.rst [hunk 2]
```diff
@@ -17,24 +17,32 @@ Required dependencies
...
+    conda install -c conda-forge mesmer
 
-and afterwards install mesmer directly from github:
+Alternately, mesmer can be installed with pip (but we make no guarantees about
+the simplicity of installing mesmer's dependencies using pip)
 
 .. code-block:: bash
 
-   python -m pip install git+https://github.com/MESMER-group/mesmer.git
+   pip install mesmer-emulator
```
Comment (mathause) [inline doc/source/installation.rst:32]:
```suggestion
   python -m pip install mesmer-emulator
```
isn't `python -m ...` the recommended way? https://docs.python.org/3/installing/index.html

File: setup.py [hunk 3]
```diff
@@ -107,6 +111,6 @@
     include_package_data=True,
     install_requires=REQUIREMENTS_INSTALL,
     extras_require=REQUIREMENTS_EXTRAS,
-    # cmdclass=cmdclass,
+    cmdclass=cmdclass,
```
Comment (mathause) [inline setup.py:114]: Interesting. Haven't seen this before.

File: CHANGELOG.rst (hunk 1 above)
Comment (znicholls) [reply CHANGELOG.rst:20]: Happy either way. I'd lean towards the full version so it renders everywhere if I had to choose.

File: doc/source/installation.rst (hunk 2 above)
Comment (znicholls) [reply doc/source/installation.rst]: I have no idea, I've always seen it written as `pip install ...` but those docs do look pretty official. Happy either way.

File: setup.py (hunk 3 above)
Comment (znicholls) [reply setup.py:120]: Ye I don't really know what it's doing, but versioneer makes you set it up...

**Review by mathause — APPROVED** [review]

**Pull request merged**